data/cube/Query.js

```diff
@@ -18,15 +18,21 @@ export class Query {
     fields;
     /** @member {CubeField[]} */
     dimensions;
-    /** @member {StoreFilter[]} */
-    filters;
     /** @member {boolean} */
     includeRoot;
     /** @member {boolean} */
     includeLeaves;
     /** @member {Cube} */
     cube;
 
+    @managed
```

Comment:
We need to apply `@ManagedSupport` to the class for this to have an effect, right?

I looked briefly to see if we could warn in the `@managed` decorator if the target wasn't marked as `isManagedSupport` but I believe that the class member decorators are being evaluated before the top-level decorator - or at least, when I logged a warning it looked to me like _all_ uses were warning with no targets having the flag set when I was attempting to run that check...